### DISCUSSION TITLE
Can not use Copilot: Extension activation failed in VS Code

### DISCUSSION THREAD

**Discussion by tiavina-mika** [community/community#114481] [New to GitHub]
`Extension activation failed: "Contact Support. You are currently logged in as tiavina-mika."`
![image](https://github.com/community/community/assets/42656064/b4e07339-edf8-4373-a680-eb4e5dc75baa)

I can not use GitHub Copilot in VS Code as usual.
May it's because my [profile ](https://github.com/tiavina-mika) is not visible to anyone, it is marked as Spam.
I can not connect to third party applications.
Please help me

**Comment by benogoulart** [comment]
Hi,

If your account has been marked as spam, that would explain both issues:

- You can't authorize third-party applications.
- GitHub Copilot in VS Code fails to authenticate and shows an activation error.

Unfortunately, this isn't something that can be resolved by reinstalling the extension or changing your VS Code settings.

I recommend checking the following:

- Verify that your email address is confirmed.
- Check whether you've received any emails from GitHub regarding account restrictions.
- If your profile is indeed flagged as spam, contact GitHub Support and request a review of your account. Only GitHub Support can remove spam-related restrictions.

Once the restriction is lifted, you should be able to authorize third-party applications and use GitHub Copilot normally again.

I hope you're able to get it resolved soon.
